hackathon_site/event/serializers.py

```diff
@@ -32,6 +32,31 @@ class Meta:
             "e_signature",
             "team",
         )
+        read_only_fields = ("id", "team", "acknowledge_rules", "e_signature")
+
+    def update(self, instance: Profile, validated_data):
+        return super().update(instance, validated_data)
+
+
+class CurrentProfileSerializer(ProfileSerializer):
+    class Meta(ProfileSerializer.Meta):
+        read_only_fields = (
+            "id",
+            "team",
+            "id_provided",
+            "attended",
```

Comment:
One last change, you'll need to add `team` to this too (since there are dedicated views with special logic for changing teams)

Reply:
Isn't team already there?

Reply:
I am completely blind apparently :man_facepalming: